examples/dt/perfscale/scalelab/README.md

```diff
@@ -0,0 +1,34 @@
+# OpenStack Scalelab Baremetal
+
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/tree/12c57baeca4ae33dd30a7707d330eb094309b4cd) on Dec 6th, 2023**
+
+This is a collection of CR templates that represent a validated Red Hat OpenStack Services on OpenShift deployment that has the following characteristics:
+
+- 3 master/worker combo-node OpenShift cluster on Baremetal
+- 3-replica Galera database
+- RabbitMQ
+- OVN networking
+- Network isolation over three NIC
+- configurable number of compute nodes based on cloud size
```

Comment:
Can you change this:
`- configurable number of compute nodes based on cloud size`
To this?
`- this example only uses two compute nodes in one node set, though it is assumed the number will be scaled`